packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1646,9 +1714,9 @@ class RoundedRectSliderTrackShape extends SliderTrackShape with BaseSliderTrackS
     assert(enableAnimation != null);
     assert(textDirection != null);
     assert(thumbCenter != null);
-    // If the slider track height is less than or equal to 0, then it makes no
-    // difference whether the track is painted or not, therefore the painting
-    // can be a no-op.
+    // If the slider [SliderThemeData.trackHeight] is less than or equal to 0,
```

Comment:
I think it may be better to actually add an "activeTrackHeight" parameter to use, but this could still work if its better documented (explain how the active track is always 2 points taller than the trackHeight, unless the trackHeight is 0)

Reply:
Went with additionalActiveTrackHeight method. Done